lib/tasks/screens.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Mix.Tasks.Addict.Gen.Screens do
+  use Mix.Task
+  import Mix.Generator
+  alias Addict.Utils
+
+  @shortdoc "Generate templates and view for Addict"
+  @moduledoc """
+  Generates an Addict view and templates
+      mix addict.gen.screens
+  """
+  def run(opts) do
+    bindings = [
+      module: Module.concat(Utils.app_module, AddictView),
+      view: Module.concat(Utils.app_module, View)
+    ]
+
+    view_file = Path.join(["web", "views", "addict_view.ex"])
+                |> Path.relative_to(Mix.Project.app_path)
+    create_file view_file, view_template(bindings)
```

Comment:
@trenpixster It's a macro that's generated by calling `embed_template :view`. You can find the documentation here: http://elixir-lang.readthedocs.org/en/latest/ref/master/mix/Mix.Generator/#Mix.Generator.embed_template/2